haystack/__init__.py

```diff
@@ -1,9 +1,9 @@
+import haystack.logging
```

Comment:
explanation for import order change here: https://github.com/deepset-ai/haystack/pull/7207/files#r1505884720